app/services/trainees/create_timeline_events.rb

```diff
@@ -73,7 +73,7 @@ def call
           ),
         ]
 
-        if hesa_or_dttp_user?
+        if hesa_or_dttp_user? && auditable_type == "Trainee"
```

Comment:
@AbigailMcP It turns out that some audits that reach this point are of type `Degree` and hence end up producing "duplicate" entries. This additional event is only for trainees I assumed.

Reply:
Makes complete sense!